src/Http/GuzzleClient.php

```diff
@@ -32,11 +33,28 @@
 
 class GuzzleClient extends Client implements ClientInterface
 {
+    /**
+     * @param array $config
+     */
+    public function __construct(array $config = [])
+    {
+        $config['oauth']['client_id']     = $config['clientId'];
```

Comment:
hm, I double checked the behavior and my assumptions were no longer correct since I last tested. I will add it to the config resolver to make these required. It will throw an exception if these are not set.

Reply:
You gotta rework some tests for that I suppose